packages/react/src/components/Footer/FooterNav.js

```diff
@@ -26,7 +26,8 @@ const FooterNav = ({ groups }) => {
   return (
     <nav
       data-autoid={`${stablePrefix}--footer-nav`}
-      className={`${prefix}--footer-nav`}>
+      className={`${prefix}--footer-nav`}
+      aria-label="Footer navigation">
```

Comment:
Do we have translations for these?

Reply:
No, from what Ram told me the role will be read by the screen reader. This is just to suppress the validation error.

Reply:
Good to know, thanks!